### ISSUE TITLE
Flatter tiles

### ISSUE THREAD

**Issue by humdingerb** [HaikuArchives/LightsOff#49]
The tiles look quite dated with the massive bevel. How about a flatter look for the GUI redesign?

![lights-off](https://user-images.githubusercontent.com/6107158/36831703-ea1ffeea-1d28-11e8-839d-0c0d9a6d5e28.png) ![lights-off_tile_mockup](https://user-images.githubusercontent.com/6107158/36831756-11502db4-1d29-11e8-855c-974c63883c21.png)

The contrast of the blue tile borders could be a bit higher, I admit...
(Could be_control_look->DrawTextControlBorder() be used?)

**Comment by owenca** [comment]
> How about a flatter look for the GUI redesign?

Yes. Also, the dark blue background is kind of depressing. Perhaps a lighter color scheme like the image below?
![lights out](https://user-images.githubusercontent.com/6434360/36885185-5d492d8a-1d9a-11e8-9551-71879add8188.png)

**Comment by humdingerb** [comment]
Using the default panel background colour is a good idea, I think. Not sure if labeled buttons are better than icon buttons. I'll playing around with mocking something up.

What'd be nice is if the tiles would re-size along with the window and board size. So the tiles of a 3x3 would be bigger, filling the available space. Possible?

**Comment by humdingerb** [comment]
How about something like this:

![lights-off_new-gui](https://user-images.githubusercontent.com/6107158/36915558-6f433910-1e51-11e8-9719-a6f740348336.png)

or:

![lights-off_new-gui-2](https://user-images.githubusercontent.com/6107158/36915921-ab0bc056-1e52-11e8-9472-63d914750509.png)

The Level pop-up could have a spinner widget, I guess...
If you wanna "mock" about some more, I can upload the wonderbrush file.

Those 3 small buttons:
* colours: change the on/off colours randomly with every click on the button.
* speaker: sound on/off
* ?: show documentation

**Comment by owenca** [comment]
I like your idea of getting rid of the menu bar. Regarding the navigation buttons, the user might confuse the "restart" button with "undo"? What about a set of 4 buttons below?

![screen shot 2018-03-04 at 12 52 53 pm](https://user-images.githubusercontent.com/6434360/36950463-1ad1e33a-1fab-11e8-9661-c9e593be35de.png)

**Comment by owenca** [comment]
> What'd be nice is if the tiles would re-size along with the window and board size. So the tiles of a 3x3 would be bigger, filling the available space. Possible?

Yes with #57.

**Comment by humdingerb** [comment]
> What about a set of 4 buttons below?

Yes. Maybe the buttons of MediaPlayer can be re-used, though it seems to draw those symbols explicitely. I'd have thought there were UTF characters for those, but can't find any in the Noto font...

Maybe try in the direction of the 2nd mockup: board, move counter, buttons. I think the buttons should stay close to the move counter as they are related in a way, and the counter shows a reaction to the use of the buttons.

**Comment by humdingerb** [comment]
> Maybe try in the direction of the 2nd mockup

I wanted to say: the 1st mockup... :)
